spec/features/users/user_updates_profile_spec.rb

```diff
@@ -15,7 +15,7 @@
     expect(page).
       to have_text(I18n.t(
         "flash.actions.update.notice",
-        resource_name: "User"
-      ))
+        resource_name: "User",
+      ),)
```

Comment:
Lastly ^

Reply:
Done in PR ad-refactor-flash-format-specs sha: 4be77c8